### DISCUSSION TITLE
How to emulate intel cpu in UTM

### DISCUSSION THREAD

**Discussion by louisliu2048** [utmapp/UTM#5805] [macOS]
My laptop is a Mac Pro M2, and I wanted to create a Linux virtual machine using Intel CPU, but I tried various configurations, including Simulate mode to create x86_64 type centos and ubuntu, but in the end I could only get AMD type CPU.

Due to project needs, I need a Linux virtual machine using Intel Cpu. I would like to know if anyone has any way to make it?

**Comment by lastCard1440** [comment]
I have the same problem on a MacBook Pro (Retina, 15-inch, Late 2013) running OS Big Sur 11.7.10. I tried to start with new device "Serial" without any success.
![2023-10-15_17-51-09](https://github.com/utmapp/UTM/assets/88321451/4c539cd7-d2da-4719-a86b-42aaff5f753b)
